atuin/src/command/client/search/interactive.rs

```diff
@@ -580,12 +580,26 @@ impl State {
             }
 
             1 => {
-                super::inspector::draw(
-                    f,
-                    results_list_chunk,
-                    &results[self.results_state.selected()],
-                    &stats.expect("Drawing inspector, but no stats"),
-                );
+                if results.is_empty() {
```

Comment:
Would it be possible to shift this logic into the inspectors draw call please? That is, within `super::inspector::draw`